plugin/entryBase.go

```diff
@@ -55,61 +55,34 @@ func NewEntry(name string) EntryBase {
 	}
 
 	e := EntryBase{
-		entryName:       name,
-		slashReplacerCh: '#',
+		name:          name,
+		slashReplacer: '#',
 	}
 	for op := range e.ttl {
 		e.SetTTLOf(defaultOpCode(op), 15*time.Second)
 	}
 	return e
 }
 
+func (e *EntryBase) partialMetadata() JSONObject {
+	if e.specifiedPartialMetadata != nil {
+		return e.specifiedPartialMetadata
+	}
+	return e.attributes.ToMap()
```

Comment:
Why do we return attributes instead?

Reply:
That's the whole "attributes subset of metadata" thing. Edge-case is if someone specifies just the attributes, but they don't set partial metadata, that likely means the attributes are the metadata. I'll add a comment.

Reply:
Updated.